src/commands/src/shared/next-gen-deploy.ts

```diff
@@ -1,38 +1,76 @@
 import { executeAndPipeCommands, executeCliCommand } from '../helpers';
 
 interface NextGenDeployParams {
-  bucket: string;
   mutPrefix: string;
   gitBranch: string;
   hasConfigRedirects: boolean;
-  url: string;
 }
 
-export async function nextGenDeploy({ bucket, mutPrefix, gitBranch, hasConfigRedirects, url }: NextGenDeployParams) {
-  if (hasConfigRedirects && (gitBranch === 'main' || gitBranch === 'master')) {
-    // equivalent to: mut-redirects config/redirects -o public/.htaccess
-    await executeCliCommand({ command: 'mut-redirects', args: ['config/redirects', '-o', 'public/.htaccess'] });
-  }
+/* This is still in development - use with caution */
+/* Logs here for future debugging purposes */
+export async function nextGenDeploy({ mutPrefix, gitBranch, hasConfigRedirects }: NextGenDeployParams) {
+  try {
+    if (hasConfigRedirects && (gitBranch === 'main' || gitBranch === 'master')) {
+      // equivalent to: mut-redirects config/redirects -o public/.htaccess
+      await executeCliCommand({ command: 'mut-redirects', args: ['config/redirects', '-o', 'public/.htaccess'] });
+      console.log(`COMMAND: mut-redirects config/redirects -o public/.htaccess`);
+    }
 
-  // equivalent to: yes | mut-publish public ${BUCKET} --prefix="${MUT_PREFIX}" --deploy --deployed-url-prefix=${URL} --json --all-subdirectories ${ARGS};
-  const { outputText } = await executeAndPipeCommands(
-    { command: 'yes' },
-    {
-      command: 'mut-publish',
-      args: [
-        'public',
-        bucket,
-        `--prefix=${mutPrefix}`,
-        '--deploy',
-        `--deployed-url-prefix=${url}`,
-        '--json',
-        '--all-subdirectories',
-      ],
-      options: {
-        cwd: `${process.cwd()}/snooty`,
-      },
+    const bucket = process.env.BUCKET;
```

Comment:
NIT: I think we should pass these values in as parameters instead of using them from the environment variable. These get set in `setupEnvironmentVariables` as a way to pass information into the makefiles, but since these values are a part of the `Job` object, we can get them from there. That way, we know where these values are coming from a bit better, and can do one less mental hoop jump to figure out how they get defined.

Reply:
I don't believe they are on the `Job`, but I do think that they could definitely be parameters for this. That's a good idea.